dev-util/poke/poke-1.4.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="extensible editor for structured binary data"
+HOMEPAGE="http://www.jemarch.net/poke"
+
+SRC_URI="mirror://gnu/poke/${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="GPL-3+"
+SLOT="0"
+IUSE="machine-interface nls static-libs test"
+
+RESTRICT="!test? ( test )"
+
+# TODO: libnbd: not packaged in gentoo yet
+# TODO: gui: does not seem to do anything :)
+RDEPEND="
+	dev-libs/boehm-gc:=
+	sys-devel/gettext
+	sys-libs/readline:=
+	machine-interface? ( dev-libs/json-c:= )
+"
+DEPEND="${RDEPEND}
+	test? ( dev-util/dejagnu )
+"
+BDEPEND="
+	sys-devel/flex
+	sys-devel/bison
+	sys-apps/help2man
+	virtual/pkgconfig
+"
+
+src_configure() {
+	econf \
+		--disable-gui \
+		--disable-libnbd \
+		$(use_enable machine-interface mi) \
+		$(use_enable nls) \
+		$(use_enable static-libs static)
```

Comment:
I suspect we don't actually want/need static-libs for this at all.

Reply:
(If you drop the line and simplify src_install, it should be fine w/ EAPI 8 which passes --disable-static)

Reply:
Thanks for the feedback. I fixed that